handler/graphql.go

```diff
@@ -40,6 +40,7 @@ type Config struct {
 	tracer                          graphql.Tracer
 	complexityLimit                 int
 	complexityLimitFunc             graphql.ComplexityLimitFunc
+	websocketOnInitFunc             func(ctx context.Context, initPayload InitPayload) bool
```

Comment:
Maybe type this func, and consider slight name change:
```suggestion
	websocketInitFunc             WebsocketInitFunc
	
	// ...
	
	type WebsocketInitFunc func(ctx context.Context, initPayload InitPayload) bool
```